jekyll/_cci2/pipelines.adoc

```diff
@@ -0,0 +1,138 @@
+
+A project can have one or more configuration sources. Configurations include, but are not limited to, a `.circleci/config.yml` file in your repository.
+
+When you create a project in CircleCI and connect it to a repository, a configuration source is set up for you automatically, and connected to a configuration file. The connected configuration file will be either a file you commit  yourself or one generated by CircleCI based on the code in your project.
+
+image::{{site.baseurl}}/assets/img/docs/project-settings-configuration.png[Configuration source]
+
+==== Add a configuration source
+
+TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a configuration source is already set up.
+
+image::{{site.baseurl}}/assets/img/docs/add-configuration-source.png[Add a configuration source]
+
+To add a configuration source, follow these steps:
+
+. In the CircleCI web app, click on **Projects** in the sidebar
+. Click on the ellipsis next to your project (icon:ellipsis-h[]) and then click **Project Settings**
+. Select **Configuration** in the sidebar
+. Click btn:[Add Configuration Source]
+. Complete the form fields and options:
+** Give your configuration source a descriptive name
+** Under Configuration Source select the platform that matches where your project is set up (for example, GitHub App)
++
+NOTE: For GitLab Self-managed, you can input any instance that you have previously set up in CircleCI. If you wish to set a different feature branch or repository from a self-managed instance as a new configuration source, you will first need to add a new connection via your xref:gitlab-integration#organization-settings-integrations[menu:Organization Settings[Integrations]]. In either case, you will also need to enter your personal access token again to authorize this connection.
+** Authorize your connection if this is not already showing with a icon:check-circle[]
+** Select your project repository
+** Select a fallback branch to use in case you try to build on a branch with no configuration defined
+** Enter the path to your configuration file relative to the root of your repository. For example, `.circleci/config.yml`
+. Click btn:[Save]
+
+Once you have set up a configuration source you need to set up a project trigger to connect it to. This is described in the following section.
+
+=== Project triggers
+
+TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a trigger is already set up.
+
+A project can have one or more triggers, which are associated with events from a source of change. Triggers include, but are not limited to, a push to a branch on a version control system (VCS), for example, a GitHub repository. A trigger determines an event (or events) along with a configuration source to define a pipeline.
```

Comment:
I think we want to surface the fact that triggers are **rules** that define what "configuration source" will be run on a given event. 

A trigger can only be associated with one config source, but a config source can have multiple triggers